ros_buildfarm/templates/status/js/setup.js

```diff
@@ -238,15 +238,24 @@ function filter_table() {
       result_rows = $.map(window.rows, function(row) {
         for (var i = 0; i < queries.length; i++) {
           var is_known_query = false;
+          var is_id_query = false;
           for(var q in QUERY_TRANSFORMS) {
-            if (queries[i] === QUERY_TRANSFORMS[q]) {
+            if (RegExp("^("+QUERY_TRANSFORMS[q]+")$").test(queries[i])) {
               is_known_query = true;
+              if (q === "id") {
```

Comment:
Currently only a single query can search for ids since it is being identified by the key. If this would check e.g. if the value starts with `id=` it would be possible to add arbitrary id -based queries in the future.